rst_files/lake_model.rst

```diff
@@ -541,7 +514,7 @@ Let's plot the path of the sample averages over 5,000 periods
   :class: test
 
   @testset begin
-      @test xbar[1] ≈ 0.043921027960428106
+      @test round(xbar[1], digits = 5) == round(0.043921027960428106, digits = 5)
```

Comment:
I think here is best to use the `atol = 1e-5`

Reply:
More importantly, why do we need to round at all?  This should be deterministic so that we can see if anything changed.  And if there is something stochastic in the test, then we should have it seeded to be deterministic anyways.

Reply:
Done.